src/plugins/saved_objects_management/public/management_section/objects_table/components/delete_confirm_modal.tsx

```diff
@@ -93,6 +102,30 @@ export const DeleteConfirmModal: FC<DeleteConfirmModalProps> = ({
             <EuiSpacer size="s" />
           </>
         )}
+        {sharedObjectsCount > 0 && (
+          <>
+            <EuiCallOut
+              data-test-subj="sharedObjectsWarning"
+              title={
+                <FormattedMessage
+                  id="savedObjectsManagement.objectsTable.deleteConfirmModal.sharedObjectsCallout.title"
+                  defaultMessage="{sharedObjectsCount, plural, one {# saved object is shared} other {# of your saved objects are shared}}"
```

Comment:
Yeah, it's worded this way on purpose. If you are only deleting one saved object, and it is shared, the table will show one object, and the callout will say `1 saved object is shared`. If we changed it to say `1 of your saved objects is shared` that would be inaccurate 🤷 @KOTungseth Gail is out atm and said to ping you for any questions -- do you have a preference? See https://github.com/elastic/kibana/issues/81605#issuecomment-860873894 for context

Reply:
@jportner I think `1 saved object is shared` is the best option here because it follows minimalism best practices, and it leaves out the personalization.